catris/server_and_client.py

```diff
@@ -58,7 +58,16 @@ def __init__(
         self.lobby: Lobby | None = None
         self.color: int | None = None
         self.view: View = AskNameView(self)
+
         self.rotate_counter_clockwise = False
+        self.lobby_id_hidden = False
+
+    def get_lobby_id_for_display(self) -> bytes:
+        if self.lobby is None:
+            return b""
```

Comment:
This doesn't make much sense, because you'll have to check for it anyway e.g. when adding `(press h to show)` message after it. It's also buggy I think.